src/modules/gfx_noisewarp.c

```diff
@@ -24,7 +24,7 @@
 
 #define FPS 30
 #define FRAMETIME (T_SECOND / FPS)
-#define FRAMES (TIME_MEDIUM * FPS) * 10
+#define FRAMES (TIME_MEDIUM * FPS)
```

Comment:
Just make this use TIME_LONG, it's exactly for this purpose.
```suggestion
#define FRAMES (TIME_LONG * FPS)
```